MatterControlLib/ApplicationView/ApplicationController.cs

```diff
@@ -1032,7 +1031,7 @@ public ApplicationController()
 
 			ProfileManager.UserChanged += (s, e) =>
 			{
-				_activePrinters = new List<PrinterConfig>();
+				//_activePrinters = new List<PrinterConfig>();
```

Comment:
TODO:  how should we handled user change in the new model?

Reply:
Ideally changing users would run the same code we execute on startup, where to look for and load the users opentabs.json and construct and load active printers